node/src/components/fetcher/trie_fetcher.rs

```diff
@@ -1 +1,341 @@
+use datasize::DataSize;
```

Comment:
Is this file restored by accident? I think Matt removed it in one of his PRs recently.